addons/binding/org.openhab.binding.windcentrale/ESH-INF/thing/millThing.xml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="windcentrale" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="mill">
+		<label>WindCentrale Wind Mill</label>
+
+		<channels>
+			<channel id="windSpeed" typeId="windSpeedType" />
+			<channel id="windDirection" typeId="windDirectionType" />
+			<channel id="powerAbsTot" typeId="powerKType">
+				<label>Total Power</label>
+			</channel>
+			<channel id="powerAbsWd" typeId="powerType">
+				<label>WD Power</label>
```

Comment:
Can you make this more clear for English users? E.g. something like "Wind share power"? The Dutch have a more describing label with "Vermogen van de winddelen" . But I can imagine the Dutch label does not fit in some UIs because of its length. :-)